### PR TITLE
added mongodb image + PHP7 driver

### PR DESCRIPTION
**Pull request by AminMkh**
Added Mongodb image + PHP7-FPM driver + php.ini configuration.

### CODE REVIEW

**Comment by AminMkh** [comment]
To test this out, should start by seeing the mongodb driver in phpinfo(),

then composer require jenssegers/mongodb": "^3.0
then configuring mongodb in config/database.php
```
            'mongodb' => [
                'driver'   => 'mongodb',
                'host'     => env('DB_HOST', 'mongo'),
                'port'     => 27017,
                'database' => env('DB_DATABASE', 'database'),
                'username' => env('DB_USERNAME', ''),
                'password' => env('DB_PASSWORD', ''),
                'options' => [
                    'database' => '' // sets the authentication database required by mongo 3
                ]
```
Then it should be set to go, using mongo with Models:
```
use Jenssegers\Mongodb\Eloquent\Model as Eloquent;
class User extends Eloquent
```

And using models normally
```
App\User::create(
	[
		'asd' => 123,
		'asd' => 2342,
	]);

$users = App\User::all();
dd($users);
```

File: php-fpm/Dockerfile-70 [hunk 1]
```diff
@@ -23,6 +23,10 @@ RUN curl -L -o /tmp/memcached.tar.gz "https://github.com/php-memcached-dev/php-m
     && docker-php-ext-install memcached \
     && rm /tmp/memcached.tar.gz
 
+# Install mongodb driver
+RUN pecl install mongodb
+RUN echo "extension=mongodb.so" > /usr/local/etc/php/conf.d/mongo.ini
```
Comment (Mahmoudz) [inline php-fpm/Dockerfile-70]: Can you try adding `extension=mongodb.so` to [laravel.ini](https://github.com/LaraDock/laradock/blob/master/php-fpm/laravel.ini) and removing it from here to see if it still work; and if it does please add another commit with this change.

**Comment by Mahmoudz** [comment]
@AminMkh another fantastic contribution, thank you.

I will merge this on Wednesday. Meanwhile can you also support the older PHP versions? If you don't have time, simply create an issue with that for anyone to join and help us with it.

Thanks again :)

File: php-fpm/Dockerfile-70 (hunk 1 above)
Comment (AminMkh) [inline php-fpm/Dockerfile-70]: sure thing, will test it out.

**Comment by Mahmoudz** [comment]
I've merged your PR a new branch called `support-mongo` https://github.com/LaraDock/laradock/tree/support-mongo and I made some commits on top of yours. 

But wasn't able to work it out!! I think it's working on your machine because you are using your machine's PHP version, where you have mongo installed and configured.

Anyhow here's my scenario:
1. I enter the workspace container and I try to install jenssegers/mongodb `composer require jenssegers/mongodb`. I get this error `Your requirements could not be resolved to an installable set of packages.`.

I guess that's because the mongo driver is not installed with PHP CLI in the workspace container.

So I guess that the missing step is installing mongo in the workspace container.

Before you start coding make sure to update your project, to get this branch and then create your PR to this branch instead of master. I will close this PR.

Thanks

**Pull request closed without merging**